pkg/agent/plugin/workloadattestor/k8s/k8s.go

```diff
@@ -195,22 +195,31 @@ func (p *Plugin) Attest(ctx context.Context, req *workloadattestorv1.AttestReque
 			return nil, err
 		}
 
+		var attestResponse *workloadattestorv1.AttestResponse
 		for _, item := range list.Items {
 			item := item
-			if isNotPod(item.UID, podUID) {
+			if podUID != "" && isNotPod(item.UID, podUID) {
```

Comment:
this podUID != "" may go into isNotPod() if not it can  affect windows code, in case in a future we are able to get podUID on windows instances